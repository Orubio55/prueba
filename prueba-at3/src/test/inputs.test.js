import { render } from "@testing-library/react";
import React from "react";

import SimpleInput from "components/inputs/simpleInput";

describe("SimpleInput", () => {
  it("should render successfully", () => {
    const { baseElement } = render(
      <SimpleInput
        type="text"
        label="Name"
        classFieldName="semi"
        value="value"
        validation={() => true}
        onChange={() => null}
      />
    );
    expect(baseElement).toBeTruthy();
  });
  it("should match snapshot", () => {
    const { baseElement } = render(
      <SimpleInput
        type="text"
        label="Name"
        classFieldName="semi"
        value="value"
        validation={() => true}
        onChange={() => null}
      />
    );
    expect(baseElement).toMatchSnapshot();
  });
});
