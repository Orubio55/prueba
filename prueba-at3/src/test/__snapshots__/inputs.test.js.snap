// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleInput should match snapshot 1`] = `
<body>
  <div>
    <span
      class="field semi"
    >
      <span
        class="control"
      >
        <div
          class="MuiFormControl-root MuiTextField-root undefined css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
        >
          <label
            class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-standard MuiFormLabel-colorPrimary MuiFormLabel-filled css-1c2i806-MuiFormLabel-root-MuiInputLabel-root"
            data-shrink="true"
            for=":r1:"
            id=":r1:-label"
          >
            Name
          </label>
          <div
            class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorPrimary MuiInputBase-formControl css-1ptx2yq-MuiInputBase-root-MuiInput-root"
          >
            <input
              aria-invalid="false"
              class="MuiInputBase-input MuiInput-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
              id=":r1:"
              style="text-align: left;"
              type="text"
              value="value"
            />
          </div>
        </div>
      </span>
    </span>
  </div>
</body>
`;
